mobile/gen_bindings.sh

```diff
@@ -45,7 +51,7 @@ protoc -I/usr/local/include -I. \
 # If prefix=1 is specified, prefix the generated methods with subserver name.
 # This must be enabled to support subservers with name conflicts.
 use_prefix="0"
-if [[ $prefix = "1" ]]
+if [ $prefix = "1" ]
```

Comment:
Actually, this needs to be quoted: `if [ "$prefix" = "1" ]` otherwise if the variable is empty, it looks like no first operand was passed in. The pleasures of shell scripts :joy:

Reply:
Ugh 🙉  Fixed